src/project/.lisa/skills/github/SKILL.md

```
@@ -398,18 +399,110 @@ lisa github sync --repo owner/repo
+```
+
+### Poll CI Until Completion
+```bash
+# Poll current branch
+lisa pr checks <PR_NUMBER>
+
+# Or use gh CLI directly
+gh pr checks <PR_NUMBER> --watch
+```
+
+### Bump Version
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)